### ISSUE TITLE
[Docs] Add ErrorHandling example in HelloMojo Notebook

### ISSUE THREAD

**Issue by maevadevs** [modularml/mojo#269]
### Where is the problem?

HelloMojo,ipynb

### What can we do better?

There isn't much explanation about the Error Handling portion.

> Both [def and fn] support raising exceptions, but this must be explicitly declared on a fn with the raises function effect, placed after the function argument list.

I think it would be helpful to add something like the simple example below for reference:

```py
fn raise_an_error(num: Int) raises -> Int:
    if num < 0:
        raise Error("I'm an error!")
    return num + 1
    

try:
    print(raise_an_error(-1))
except:
    print("exception")
```

**Comment by scottamain** [comment]
Thanks for this suggestion! There are definitely some basic language features not documented yet, but these sort of things should typically work the same as in Python, so they have been lower on our documentation priorities. We'll get to it soon, though. Thanks again.

**Comment by Mogball** [comment]
@scottamain any status update on this?

**Comment by scottamain** [comment]
Sorry, other priorities continue to keep this one low. I believe all error handling in Mojo is the same as Python's, right @Mogball?

**Comment by Mogball** [comment]
Yes. It's the same. If you are not actively working on it, please unassign yourself

**Comment by scottamain** [comment]
Sorry for the long silence here. This page was replaced a while ago with a new Mojo Get Started guide. It doesn’t show error handling exactly but does discuss it briefly [in this section](https://docs.modular.com/mojo/manual/get-started#9-implement-support-for-the-stringableraising-trait) with links to the separate [docs about error handling](https://docs.modular.com/mojo/manual/errors) (which is newer than this issue).
